OpenUpSA/wazimap-ng-ui
# fixing the bug that breaks the site when a ward is selected

## Description
investigating and fixing the bug that breaks the site when a ward is selected

## Related Issue
https://trello.com/c/dSHDxoSs/864-site-breaks-when-i-select-a-ward

## How to test it locally
On sanef profile

* select johannesburg
* select a ward
* open the rich data view
* confirm that the ward data is displayed instead of johannesburg and there is no console error

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/127233687-9e5df225-f429-4995-9ac6-6ba5f89ee796.png)


## Changelog

### Added

### Updated
* `src/js/setup/dataexplorer.js` to correct the variable name

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
